pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotQueryResource.java

```diff
@@ -121,9 +121,17 @@ public String handleGetSql(@QueryParam("sql") String sqlQuery, @QueryParam("trac
   }
 
   private String executeSqlQuery(@Context HttpHeaders httpHeaders, String sqlQuery, String traceEnabled,
-      String queryOptions)
+      @Nullable String queryOptions)
```

Comment:
Seems this is the only required change to be able to parse query options from the query?